RecoParticleFlow/PFClusterTools/test/ProducePFCalibration.py

```diff
@@ -19,69 +19,95 @@
 process.mywriter = cms.EDAnalyzer(
   "ProducePFCalibrationObject",
   write = cms.untracked.bool(False),
-  toWrite = cms.VPSet(
-            cms.PSet(fType      = cms.untracked.string("PFfa_BARREL"), 
-                     formula    = cms.untracked.string("[0]+([1]+[2]/sqrt(x))*exp(-x/[3])-[4]*exp(-x*x/[5])"),
+ toWrite = cms.VPSet(
+            cms.PSet(fType      = cms.untracked.string("PFfa_BARREL"),
+                     formula    = cms.untracked.string("[0]+((([1]+([2]/sqrt(x)))*exp(-(x^[6]/[3])))-([4]*exp(-(x^[7]/[5]))))"),
                      limits     = cms.untracked.vdouble(1., 1000.),
-                     parameters = cms.untracked.vdouble(1.15665, 0.165627, 0.827718, 231.339, 2.45332, 29.6603,  ) 
+                     parameters = cms.untracked.vdouble(-13.9219, 14.9124, 5.38578, 0.861981, -0.00759275, 3.73563e-23, -1.17946, -13.3644)
                     ),
-            cms.PSet(fType      = cms.untracked.string("PFfb_BARREL"), 
-                     formula    = cms.untracked.string("[0]+([1]+[2]/sqrt(x))*exp(-x/[3])-[4]*exp(-x*x/[5])"),
+            cms.PSet(fType      = cms.untracked.string("PFfb_BARREL"),
+                     formula    = cms.untracked.string("[0]+((([1]+([2]/sqrt(x)))*exp(-(x^[6]/[3])))-([4]*exp(-(x^[7]/[5]))))"),
                      limits     = cms.untracked.vdouble(1., 1000.),
-                     parameters = cms.untracked.vdouble(0.994603, 0.13632, -0.758013, 183.627, 1, 39.6784,  ) 
+                     parameters = cms.untracked.vdouble(2.18802, 0.522053, -4.67354, 1.2109e+06, 1.75318, 0.187919, -9241.43, -0.607392, )
```

Comment:
in order to avoid copy-paste mistakes, it appears important to make all parameters in this file to be readable by PFEnergyCalibration at construction step. So that the functions used there for non-DB inputs can be parsed from the same place.

To highlight this I note that the parameters here and in the updated fbBarrel TF1 do not match:
- p3 is 1.2109e+06 vs 12.109 ("e+06" is "lost")
- p6 is -9241.43 here vs 6.26234 there